core/src/main/scala/com/evolutiongaming/catshelper/Log.scala

```diff
@@ -134,6 +134,38 @@ object Log {
     }
   }
 
+  def apply[F[_] : Sync](logger: ch.qos.logback.classic.Logger): Log[F] = new Log[F] {
+
+    import ch.qos.logback.classic.Level
+    import ch.qos.logback.classic.spi.LoggingEvent
+
+    import scala.collection.JavaConverters._
+
+    val FQCN = getClass.getName
+
+    def append(msg: => String, mdc: Mdc, level: Level, throwable: Throwable = null): F[Unit] = Sync[F].delay {
+      val event = new LoggingEvent(FQCN, logger, level, msg, throwable, null)
+      mdc.context.map(_.toSortedMap) foreach { mdc =>
```

Comment:
could it be too expensive to call `toSortedMap` on every log call?

Reply:
what if logging is disabled ?

Reply:
Shouldn't be expensive as it's just cast: `asInstanceOf`